python-packages/order_utils/setup.py

```diff
@@ -0,0 +1,119 @@
+"""setuptools module for order_utils package."""
+
+import subprocess  # nosec
+from shutil import rmtree
+from os import path, remove, walk
+
+from distutils.command.clean import clean  # type: ignore
+from setuptools import setup  # type: ignore
+import setuptools.command.build_py  # type: ignore
+from setuptools.command.test import test as TestCommand  # type: ignore
```

Comment:
Are you ignoring the type checks because these packages don't have any?

Reply:
yes.  or, maybe it's just that i don't have them installed/available in my environment?  this will evolve when i start actually coding with types and understand how the distribution/packaging of types works.